Src/DataTypes/Data.cs

```diff
@@ -129,7 +111,11 @@ public static Data CreateDatasource(string type, string path, int batchSize)
 
             if (!data.Exists())
             {
-                throw new Exception($"path '{path}' to data does not exist.");
+                //skip exsistence validation for base data to allow creation of new data for FS and S3
+                if (isBase)
+                    Console.WriteLine($"base data at path '{path}' does not exists and will be created");
```

Comment:
Did you make sure it is really created? The current logic is supposed to support only the case in which the directory already exists.

Reply:
it worked for me with fs,
cant test with s3

Reply:
fixed for fs,
should work for s3 but I cant test it

Reply:
tough this case cause validation to fail